### ISSUE TITLE
No longer able to annotate framework files

### ISSUE THREAD

**Issue by fourman-alex** [flutter/flutter-intellij#5368]
## Steps to Reproduce
While working on a flutter project and viewing a flutter SDK file, for example, `framework.dart` it is no longer possible to select annotate or show history of that file. The current workaround would be to open the flutter SDK project itself, navigate to that file and only then annotate would become available. 
This seems to be a regression as I clearly remember it working before I just can't say for sure when the regression was introduced.

Flutter plugin version is: 54.1.1
Dart plugin version is: 201.9335
Even though flutter doctor doesn't report them for android studio for some reason

## Version info

```
[✓] Flutter (Channel stable, 2.0.3, on Linux, locale en_US.UTF-8)
    • Flutter version 2.0.3 at /home/alex/dev/flutter
    • Framework revision 4d7946a68d (2 weeks ago), 2021-03-18 17:24:33 -0700
    • Engine revision 3459eb2436
    • Dart version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /home/alex/Android/Sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: /home/alex/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at google-chrome

[✓] Android Studio
    • Android Studio at /home/alex/android-studio
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] IntelliJ IDEA Community Edition (version 2020.1)
    • IntelliJ at /home/alex/idea-IC-201.7223.91
    • Flutter plugin version 45.1.3
    • Dart plugin version 201.7223.99

[✓] VS Code (version 1.52.1)
    • VS Code at /usr/share/code
    • Flutter extension version 3.20.0

[✓] Connected device (2 available)
    • sdk gphone x86 64 (mobile) • emulator-5554 • android-x64    • Android 11 (API 30) (emulator)
    • Chrome (web)               • chrome        • web-javascript • Google Chrome 89.0.4389.90

• No issues found!
```

**Comment by stevemessick** [comment]
@fourman-alex Are you referring to the git operations `Annotate with Git Blame` and `Show History`?

**Comment by fourman-alex** [comment]
@stevemessick Yes, that is exactly that.

**Comment by stevemessick** [comment]
Git operations are only available on files that are checked out from a git repository. The Flutter SDK is not distributed as a repository, but you are able to check it out from github.

**Comment by fourman-alex** [comment]
@stevemessick I did check out the SDK from GitHub the same way I check out any project. 
I cloned it with: `git__ clone https://github.com/flutter/flutter.git`
This repository is can be annotated if I open it independently, the issue I'm reporting is that it can't be annotated while viewing it from another flutter project that uses that exact repository as the Flutter SDK location.

I think you closed this issue prematurely. Is there any additional information I can add to better demonstrate the issue?
